onos_ric_sdk_py/sdl.py

```diff
@@ -193,13 +199,22 @@ async def set_cell_data(
         except GRPCError as e:
             raise ClientRuntimeError() from e
 
-        if data is None:
-            if key not in resp.object.aspects:
-                return
+        op_count = 0
+        for key, data in key_data_map.items():
+            if data is None:
+                if key not in resp.object.aspects:
+                    pass
+                else:
+                    del resp.object.aspects[key]
+                    op_count += 1
             else:
-                del resp.object.aspects[key]
-        else:
-            resp.object.aspects[key] = betterproto.lib.google.protobuf.Any(key, data)
+                resp.object.aspects[key] = betterproto.lib.google.protobuf.Any(
+                    key, data
+                )
+                op_count += 1
+
+        if op_count == 0:
+            return None
```

Comment:
Does this need to be `return None` or will `mypy` complain about a bare `return` ?

Reply:
oh yeah `return` totally makes sense. mypy is still happy

Reply:
How would the RIC know which e2node it's operating on without `e2_node_id` as a parameter?